src/skmatter/sample_selection/_base.py

```diff
@@ -15,7 +15,7 @@
 
 def _linear_interpolator(points, values):
     """
-    Returns linear interpolater for unstructured D-D data. Tessellate the input point
+    Returns linear interpolator for unstructured D-D data. Tessellate the input point
```

Comment:
I typically don't like multiple types of changes within one PR, can you roll these into a separate one?

Reply:
I am all for consistency but I'm not going to make a separate pull request to correct a couple of typos.